packages/app/src/AppRoutes.tsx

```diff
@@ -70,14 +71,6 @@ export default function AppRoutes() {
             </RequireWallet>
           }
         />
-        <Route
```

Comment:
This faucet page cannot be removed, if you enable it on your env; `REACT_APP_ENABLE_FAUCET_API=true` you should be able to see this screen. It's a faucet with captcha.

Reply:
I think we forgot to create a task to update this screen

Reply:
✅